proto/controlclient.proto

```diff
@@ -37,3 +39,20 @@ message _ListCachesResponse {
   repeated _Cache cache = 1;
   string next_token = 2;
 }
+
+message _CreateSigningKeyRequest {
+  int32 ttl_minutes = 1;
```

Comment:
@kvcache - I think ttl in minutes here is ok - just wanted to get that confirmation from you

Reply:
Yeah, that seems pragmatic.

BUT

why not uint32?

Reply:
Good catch - we should change that here and in the internal proto as well - https://github.com/momentohq/kotlin-kitchensink/pull/160/files